### PR TITLE
Fix to clarify that an X.509 SVID must contain one URI SAN

### PR DESCRIPTION
**Pull request by amartinezfayo**
Change  from "An X.509 SVID MUST NOT contain more than one URI SAN" to "An X.509 SVID MUST contain one URI SAN"

### CODE REVIEW

File: standards/X509-SVID.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Perhaps the most important function of SPIFFE is to secure process to process co
 This specification addresses the encoding of SVID information into an X.509 certificate, the constraints which must be set, as well as how to validate X.509 SVIDs.
 
 ## 2. SPIFFE ID
-In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST NOT contain more than one URI SAN. It MAY contain any number of other SAN fields, including DNS SANs.
+In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST contain one URI SAN. It MAY contain any number of other SAN fields, including DNS SANs.
```
Comment (evan2645) [inline standards/X509-SVID.md]: Perhaps we can change this to `MUST contain exactly one` or `MUST contain only one`? In the proposed wording, I feel that the "one and only one" semantics are not clear

Comment (amartinezfayo) [reply standards/X509-SVID.md]: I agree. I think that `MUST contain exactly one` is more clear. I'll change to that.

**Review by evan2645 — APPROVED** [review]
LGTM

This change needs at least one additional approver

**Comment by dudedad** [comment]
LGTM.  Agreed.

**Review by mattmoyer — APPROVED** [review]

**Pull request merged**